conjur/data_object/OsTypes.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class OsTypes(Enum):
```

Comment:
Needs docstrings and I think the name should be `OSTypes` otherwise looks weird IMO 🤷‍♀️

Reply:
why is this under data_objects folder? Don't you think it should be under `util`?

Reply:
@mbenita-Cyberark